pkg/types/aws/validation/platform.go

```diff
@@ -23,12 +23,15 @@ var (
 		"cn-north-1":     "Beijing",
 		"cn-northwest-1": "Ningxia",
 		"eu-central-1":   "Frankfurt",
+		"eu-north-1":     "Stockholm",
 		"eu-west-1":      "Ireland",
 		"eu-west-2":      "London",
 		"eu-west-3":      "Paris",
 		"sa-east-1":      "São Paulo",
 		"us-east-1":      "N. Virginia",
 		"us-east-2":      "Ohio",
+		"us-gov-east-1":  "AWS GovCloud (US-East)",
+		"us-gov-west-1":  "AWS GovCloud (US-West)",
```

Comment:
You sure about this? The pricing API calls this `AWS GovCloud (US)` I think...

Reply:
> The pricing API calls this `AWS GovCloud (US)` I think...

It does, but I have an entry for decoding that [here][1].  They use `US-West` in other places, and now that `US-East` exists, I think the qualified name makes more sense here.

[1]: https://github.com/openshift/installer/pull/1163/files#diff-2a3e5e086f283b55d7555e6b024479a1R76